tests/io/aims/test_aims_parsers.py

```diff
@@ -0,0 +1,1157 @@
+    assert np.allclose(calc_chunk.results["hirshfeld_charges"], hirshfeld_charges)
+
+
+def test_calc_hirshfeld_volumes(calc_chunk):
+    hirshfeld_volumes = [73.39467444, 62.86011074]
+    assert np.allclose(calc_chunk.hirshfeld_volumes, hirshfeld_volumes)
+    assert np.allclose(calc_chunk.results["hirshfeld_volumes"], hirshfeld_volumes)
+
+
+def test_calc_hirshfeld_atomic_dipoles(calc_chunk):
+    hirshfeld_atomic_dipoles = np.zeros((2, 3))
+    assert np.allclose(calc_chunk.hirshfeld_atomic_dipoles, hirshfeld_atomic_dipoles)
+    assert np.allclose(calc_chunk.results["hirshfeld_atomic_dipoles"], hirshfeld_atomic_dipoles)
+
+
+def test_calc_hirshfeld_dipole(calc_chunk):
+    assert calc_chunk.hirshfeld_dipole is None
+
+
+@pytest.fixture
+def molecular_header_chunk():
+    lines = """
+        | Number of atoms                   :        3
+        | Number of spin channels           :        1
+        The structure contains        3 atoms  and a total of         10.000 electrons.
+        Input geometry:
+        | Atomic structure:
+        |       Atom                x [A]            y [A]            z [A]
+        |    1: Species O             0.00000000        0.00000000        0.00000000
+        |    2: Species H             0.95840000        0.00000000        0.00000000
+        |    3: Species H            -0.24000000        0.92790000        0.00000000
+        'Geometry relaxation: A file geometry.in.next_step is written out by default after each step.'
+        | Maximum number of basis functions            :        7
+        | Number of Kohn-Sham states (occupied + empty):       11
+        Reducing total number of  Kohn-Sham states to        7.
+    """
+
+    lines = lines.splitlines()
+    for ll, line in enumerate(lines):
+        lines[ll] = line.strip()
+
+    return AimsOutHeaderChunk(lines)
+
+
+@pytest.mark.parametrize(
+    "attrname",
+    [
+        "k_points",
+        "k_point_weights",
+        "initial_lattice",
+        "n_k_points",
+    ],
+)
+def test_chunk_molecular_header_defaults_none(attrname, molecular_header_chunk):
+    assert getattr(molecular_header_chunk, attrname) is None
+
+
+def test_molecular_header_n_bands(molecular_header_chunk):
+    assert molecular_header_chunk.n_bands == 7
+
+
+def test_molecular_header_initial_structure(molecular_header_chunk, molecular_positions):
+    assert len(molecular_header_chunk.initial_structure) == 3
+    assert np.all(["O", "H", "H"] == [sp.symbol for sp in molecular_header_chunk.initial_structure.species])
+    assert np.allclose(
+        molecular_header_chunk.initial_structure.cart_coords,
+        np.array(
+            [
+                [0.00000000, 0.00000000, 0.00000000],
+                [0.95840000, 0.00000000, 0.00000000],
+                [-0.24000000, 0.92790000, 0.00000000],
+            ]
+        ),
+    )
+
+
+@pytest.fixture
+def molecular_calc_chunk(molecular_header_chunk):
+    lines = """
```

Comment:
Same here.

Reply:
moved all lines into parser files that are gzipped